packages/mobile/src/components/AccountOverview.tsx

```diff
@@ -90,7 +91,7 @@ export class AccountOverview extends React.Component<Props> {
                 style={[style.education, !this.props.stableEducationCompleted && style.dotOffset]}
               >
                 <CurrencyDisplay
-                  amount={dollarBalance}
+                  amount={new BigNumber(dollarBalance || 0)}
```

Comment:
This makes `<CurrencyDisplay />` re-render every time this component is rendered. I don't think it's a problem. just noting it.